lib/_stream_writable.js

```diff
@@ -372,6 +377,7 @@ function writeOrBuffer(stream, state, chunk, encoding, cb) {
 }
 
 function doWrite(stream, state, writev, len, chunk, encoding, cb) {
+  debug('Writting');
```

Comment:
`debug('Writing %o', chunk);` would be more useful

Reply:
Thank you sr, fixing this!